ts/webworker/workers/browser/libsession_worker_interface.ts

```diff
@@ -225,7 +225,21 @@ const groups: Map<GroupPubkeyType, UserGroupsGet> = new Map();
 
 function dispatchCachedGroupsToRedux() {
   window?.inboxStore?.dispatch?.(
-    userGroupsActions.refreshUserGroupsSlice({ groups: [...groups.values()] })
+    userGroupsActions.refreshUserGroupsSlice({
+      groups: [...groups.values()].map(makeUserGroupGetRedux),
```

Comment:
So we are loading groups initially with partial information that is then stored in redux for performance? I assume this is to speed up initial convo loading? 

I have assumed we only load partial information because you use the `pick` type in the redux group slice.

Reply:
yes, we load and cache what we need for classic redux auto rerendering